home_assignment/controllers/messages_routes.py

```diff
@@ -0,0 +1,76 @@
+# Import the framework
+from flask import Flask, request, json
+from flows.delete_message import DeleteMessageFlow
+from flows.get_all_unread_user_messages import UnreadMessageFlow
+from flows.get_all_user_messages import ReadMessageFlow
+from flows.read_one_message import ReadOneMessageFlow
+from flows.send_message import SendMessageFlow
+
+
+def router(app):
+    # @app.route("/example", methods=["GET"])
+    # def example():
+    #     return json_dumps({"This Is": "An example"})
+    #
+    # @app.route("/do", methods=["POST"])
+    # def process_application():
+    #     request_data = json_loads(request.get_json())
+    #     my_param_key = request_data["my_param_key"]
+    #     return json_dumps({"message": f"Done with {my_param_key}"})
+
+    @app.route('/messages/write', methods=['POST'])
+    def send_message() -> str:
+        request_data = request.get_data()
+        data = json.loads(request_data)
+        user_id = data.get('user_id')
+        name = data.get('name')
+        title = data.get('title')
+        message = data.get('message')
+        read = False
+        created_at = data.get('created_at')
+
+        flow = SendMessageFlow("Test1")
+        result = flow.send(user_id, name, title, message, read, created_at)
+        if user_id and name and title and message:
+            return {"New message": f"id: {result.user_id}, "
+                                   f"name: {result.name}, "
+                                   f"title: {result.title}, "
+                                   f"message: {result.message}, "
+                                   f"read: {result.read}, "
+                                   f"created_at: {result.created_at} "
```

Comment:
Think about extendability and SOLID.
Why do you need to write it again and again?

Reply:
DONE.